crocket/src/main/java/com/crocket/Application.java

```diff
@@ -23,30 +24,38 @@ public static void main( String[] args )
 
     private static void init() {
         CroquetView frame = CroquetView.getInstance();
-        Level level1 = new Level1();
-        LevelView level1View = new LevelView();
+        // Level level1 = new Level1();
+        // LevelView level1View = new LevelView();
+        Level levelTest = new LevelTest();
```

Comment:
Why leave this comments?

Reply:
Easily switch between the two levels, we don't really have time to implement a way for the user to manually choose.